### PR TITLE
Saving user credentials in shared prefs

### PR DESCRIPTION
**Pull request by AdamGrzybkowski**
After succesful login user's credentials are stored in shared preferences. We use it later to check if user is logged in or not. 

TODO: Encrypt user password sotre in shared prefs ?

### CODE REVIEW

**Comment by pkprzekwas** [comment]
It would be nice to store hash instead of password string in database, although it is not 100% safe as well :)

File: app/src/main/java/com/gut/follower/activities/SplashActivity.java [hunk 1]
```diff
@@ -3,16 +3,23 @@
...
-        finish();
+        if(SessionManager.isUserLoggedIn(getApplicationContext())){
+            Intent intent = new Intent(getApplicationContext(), MainActivity.class);
+            intent.setFlags(Intent.FLAG_ACTIVITY_NEW_TASK | Intent.FLAG_ACTIVITY_CLEAR_TASK);
+            startActivity(intent);
+        } else {
+            Intent intent = new Intent(getApplicationContext(), LoginActivity.class);
+            intent.setFlags(Intent.FLAG_ACTIVITY_NEW_TASK | Intent.FLAG_ACTIVITY_CLEAR_TASK);
+            startActivity(intent);
+        }
```
Comment (pkprzekwas) [inline app/src/main/java/com/gut/follower/activities/SplashActivity.java:18]: You repeat yourself here and in some other places. Try to consider excluding this three lines of code (maybe even to BaseActivity) as a method and send activity type as an argument.

**Review by pkprzekwas — COMMENTED** [review]
From what I understands it looks good for me. Feel free to consider my suggestion and merge.

File: app/src/main/java/com/gut/follower/activities/SplashActivity.java (hunk 1 above)
Comment (AdamGrzybkowski) [reply app/src/main/java/com/gut/follower/activities/SplashActivity.java:18]: Good idea :) Done.

**Comment by AdamGrzybkowski** [comment]
@pkprzekwas merged this, we can think later about saving password :)

**Pull request merged**